include/pnet_api.h

```diff
@@ -32,7 +32,9 @@
 extern "C" {
 #endif
 
-#include <pnet_export.h>
+#include "pnet_export.h"
```

Comment:
Shouldn't it be possible to use <> here as the files are in the include-directory?

Reply:
Probably, but it depends on how the user installs the include files and how they configure their include search path. Whereas "" always works, we want to include a file we know is in the same directory.